src/test/java/org/zaproxy/zap/extension/hud/ui/uimap/HUD.java

```diff
@@ -46,12 +46,17 @@ public HUD(WebDriver webdriver) {
         this.webdriver = webdriver;
     }
 
-    public static By LEFT_PANEL_BY_ID = By.id("left-panel");
-    public static By RIGHT_PANEL_BY_ID = By.id("right-panel");
-    public static By BOTTOM_PANEL_BY_ID = By.id("bottom-drawer");
-    public static By DISPLAY_PANEL_BY_ID = By.id("main-display");
+    public static By LEFT_PANEL_BY_ID = By.id("zap-hud-left-panel");
+    public static By RIGHT_PANEL_BY_ID = By.id("zap-hud-right-panel");
+    public static By BOTTOM_PANEL_BY_ID = By.id("zap-hud-bottom-drawer");
+    public static By DISPLAY_PANEL_BY_ID = By.id("zap-hud-main-display");
+
+    public static By HISTORY_TABLE_BY_ID = By.id("history-messages");
+
     public static By HUD_BUTTON_BY_CLASSNAME = By.className("hud-button");
+
     public static int LEFT_PANEL_MIN_BUTTONS = 8;
+    public static String HISTORY_TAB_LABEL = "History";
```

Comment:
Should this have some sort of i18n treatment?

Reply:
This is just for the tests, so as long as we just run them in English (the default) then we should be fine.
Obviously it would be great to have tests for the i18n as well...